### DISCUSSION TITLE
Show Copyq main window by shortcut icon on desktop

### DISCUSSION THREAD

**Discussion by Alexnoj** [hluk/CopyQ#2346] [General]
I saw that option show the window by keyboard shortcut. but cant find option open copyq from icon on desktop.
* WINDOWS 10

**Comment by hluk** [comment]
I do not use Windows and cannot test this, but it should be possible to modify the desktop icon to show the main window (and start the app if not started yet) by adding additional arguments to `copyq.exe`: `--start-server show`
